test/workbox-background-sync/node/lib/test-Queue.mjs

```diff
@@ -0,0 +1,535 @@
+/*
+ Copyright 2017 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import {expect} from 'chai';
+import clearRequire from 'clear-require';
+import sinon from 'sinon';
+import expectError from '../../../../infra/testing/expectError';
+import indexedDBHelper, {DBWrapper} from
+    '../../../../packages/workbox-core/utils/indexedDBHelper.mjs';
+
+
+let Queue;
+
+/**
+ * Suspends execution in an async function for the specified amount of time.
+ *
+ * @param {number} time
+ * @return {Promise}
+ */
+const sleep = (time) => new Promise((resolve) => setTimeout(resolve, time));
+
+
+const clearObjectStore = async () => {
+  const db = await indexedDBHelper.getDB(
+      'workbox-background-sync', 'requests', {autoIncrement: true});
+
+  const items = await db.getAll();
+  for (const [key] of items.entries()) {
+    await db.delete(key);
+  }
+};
+
+
+describe(`backgroundSync.Queue`, function() {
+  const sandbox = sinon.sandbox.create();
+
+  beforeEach(async function() {
+    sandbox.restore();
+    clearRequire.match(RegExp('workbox-background-sync/lib/Queue.mjs'));
```

Comment:
add a comment that this is to clear the names in the Queue.mjs.

Reply:
Done.